README.md

```diff
@@ -43,6 +43,153 @@ The documentation is automatically generated at `/docs` and `/redoc`. For produc
 
 [WIP]
 
+# Instalación y Configuración de LocalStack
+
+Este documento describe los pasos necesarios para instalar y configurar LocalStack, así como las herramientas necesarias para su correcto funcionamiento.
+
+## Requisitos previos
+
+Antes de comenzar con la instalación de LocalStack, asegúrate de tener instalados los siguientes componentes en tu máquina:
+
+1. **AWS CLI**: Para la administración de scripts de AWS y LocalStack.
+   
+   Instala AWS CLI ejecutando el siguiente comando:
+   ```bash
+   curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+   sudo installer -pkg AWSCLIV2.pkg -target /
+    ```
+   
+2. **Configura AWS CLI con las credenciales para LocalStack:**
+
+   ```bash
+   aws configure --profile localstack
+   # Introduce las siguientes credenciales
+   AWS Access Key ID [None]: test
+   AWS Secret Access Key [None]: test
+   Default region name [None]: us-east-1
+   Default output format [None]: json
+   ```
+   
+3. **Docker:** LocalStack se ejecuta en contenedores Docker, por lo que es necesario tener Docker instalado. Descárgalo e instálalo desde Docker.
+
+4. **jq:** Una herramienta de línea de comandos para procesar JSON, utilizada en los scripts de despliegue. Si no está instalado, puedes hacerlo ejecutando:
+
+    ```bash
+    sudo apt-get install -y jq  # Para Linux
+    brew install jq             # Para macOS
+    ```
+   
+# Instalación de LocalStack
+
+**Paso 1:** Crear los paquetes para Lambda
+Antes de iniciar LocalStack, es necesario empaquetar los archivos de los requisitos para las funciones Lambda.
+
+Ejecuta el siguiente comando para comprimir los paquetes light y heavy requeridos para el despliegue de Lambda:
+
+   ```bash
+   python package_lambda.py
+   ```
+
+**Paso 2:** Iniciar los servicios con Docker Compose
+Inicia los servicios de LocalStack y PostgreSQL utilizando Docker Compose:
+
+```bash
+  docker-compose up
+```
+
+**Paso 3:** Desplegar LocalStack
+Una vez que los contenedores estén en ejecución, ejecuta el script de despliegue `deploy-local.sh` para configurar el entorno:
+
+```bash
+  ./deploy-local.sh
```

Comment:
To execute this script, it is necessary to set the appropriate execution permissions to file. Please include a step for this process.